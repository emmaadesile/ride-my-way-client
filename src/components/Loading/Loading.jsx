import React from "react";

const Loading = () => (
  <div className="loading-container">
    <section className="text-center">
      <span className="loading__animation loading" />
    </section>
  </div>
);

export default Loading;
