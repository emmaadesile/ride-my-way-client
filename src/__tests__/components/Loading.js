import React from "react";
import { shallow } from "enzyme";
import Loading from "../../components/Loading/Loading";

describe("Loading component", () => {
  test("renders the Loading component", () => {
    const wrapper = shallow(<Loading />);
    expect(wrapper.exists()).toBe(true);
  });
});
