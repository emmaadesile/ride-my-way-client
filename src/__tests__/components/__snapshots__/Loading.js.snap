// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should render correctly 1`] = `
<div
  className="loading-container"
>
  <section
    className="text-center"
  >
    <span
      className="loading__animation loading"
    />
  </section>
</div>
`;
